test/logger.spec.js

```diff
@@ -31,6 +31,15 @@ describe('logger', () => {
     return consoleOptions;
   }
 
+  it('should configure logColor as boolean in minimist', () => {
+    mock('minimist', (args, opts) => {
+      expect(opts.boolean).toEqual('logColor');
+      return {};
+    });
+
+    mock.reRequire('../src/logger');
+  });
+
   it('should accept the logColor flag', () => {
     const opts = setupTest({ logColor: false });
```

Comment:
What happens in your unit test if you set it as a string? Might be good to test that aspect?

```
const opts = setupTest({ logColor: 'false' });
```

Reply:
Good question.  Setting `boolean: logColor` to the minimist options means `logColor` will always be a boolean.